lib/modules/hosts/pastebin.js

```diff
@@ -1,5 +1,12 @@
 modules['showImages'].siteModules['pastebin'] = {
 	domains: ['pastebin.com'],
+	options: {
+		width: {
+			description: 'The maximum width of pastebin expandos. (any valid CSS width, default 80em)',
```

Comment:
whenever the options GUI get redesign, we should throw in a place to show the default value more obviously t han hovering over the title